.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# Learn about CODEOWNERS file format:
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @open-telemetry/cpp-contrib-approvers
+
+instrumentation/httpd/* @TomRoSystems @open-telemetry/cpp-contrib-approvers
+instrumentation/nginx/* @seemk @open-telemetry/cpp-contrib-approvers
```

Comment:
The problem with this is that `*` at the end of path will not match subdirectories.

Also I was thinking about some extra rule for CODEOWNERS file itself? I was not aware of this PR and created https://github.com/open-telemetry/opentelemetry-cpp-contrib/pull/13 but please close it was this one is merged.